### ISSUE TITLE
--images flag ignored for some components

### ISSUE THREAD

**Issue by thoraxe** [openshift/origin#936]
Given the following master options:

```
OPTIONS="--loglevel=4 --public-master=ose3-master.example.com --images=openshift3_beta/ose-${component}"
```

And given the availability of the image:

```
[root@ose3-master ~]# docker images | grep pod
openshift/origin-pod                                                                 latest              1f4043fc537f        13 hours ago        957.8 kB
openshift3_beta/ose-pod                                                              latest              e55ad09234ce        17 hours ago        955 kB
docker-buildvm-rhose.usersys.redhat.com:5000/openshift3_beta/ose-pod                 latest              e55ad09234ce        17 hours ago        955 kB
```

We can see that openshift/origin-pod still was fetched and was used:

````
[root@ose3-master ~]# docker ps
CONTAINER ID        IMAGE                                                                                    COMMAND                CREATED             STATUS              PORTS                                      NAMES
d6448b1c0a8c        openshift/hello-openshift:latest                                                         "/hello_openshift"     3 minutes ago       Up 3 minutes                                                   k8s_hello-openshift.e220ea5e_hello-openshift.default.api_4dca6030-ae03-11e4-a3cc-525400b33d1d_83e4f98d               
137407e5439a        openshift/origin-pod:latest                                                              "/pod"                 4 minutes ago       Up 4 minutes        0.0.0.0:6061->8080/tcp                     k8s_POD.c922de90_hello-openshift.default.api_4dca6030-ae03-11e4-a3cc-525400b33d1d_2811a558                           
98ffbc2d1ac5        docker-buildvm-rhose.usersys.redhat.com:5000/openshift3_beta/ose-haproxy-router:latest   "/usr/bin/openshift-   5 minutes ago       Up 5 minutes                                                   k8s_origin-haproxy-router-mainrouter.56c65708_mainrouter.default.api_1bab8c43-ae03-11e4-a3cc-525400b33d1d_6cb50b02   
595ae02f7090        openshift/origin-pod:latest                                                              "/pod"                 5 minutes ago       Up 5 minutes        0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp   k8s_POD.f9e40051_mainrouter.default.api_1bab8c43-ae03-11e4-a3cc-525400b33d1d_aa209b8e 
```

Unless I am doing something wrong in my incantation, it would seem that the flag is being ignored (at least for origin-pod).

**Comment by thoraxe** [comment]
Looks like it needs to be specified for the node as well.

**Comment by smarterclayton** [comment]
The help on the cli probably needs to be updated

**Comment by smarterclayton** [comment]
Help in CLI fixed in #940
